### PR TITLE
Added docstrings to BlueHatGreenHat.py, animal.py, clothing.py.

### PR DESCRIPTION
**Pull request by tim-andes**
See title

### CODE REVIEW

File: BlueHatGreenHat/BlueHatGreenHat.py [hunk 1]
```diff
@@ -3,8 +3,16 @@
```
Comment (Meltz014) [inline BlueHatGreenHat/BlueHatGreenHat.py:12]: Can you add a module level docstring here that depicts the goal for the program?  I.e. to generate the text of Blue Hat Green Hat by sandra boynton, etc.

File: BlueHatGreenHat/animal.py [hunk 2]
```diff
@@ -70,6 +77,7 @@ def __init__(self):
                       self.hands]
 
     def swim(self):
+        """Upon a swim, any current clothing raises OopsError('Oops!')."""
```
Comment (Meltz014) [inline BlueHatGreenHat/animal.py:91]: Swim naked, baby

File: BlueHatGreenHat/clothing.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 class Clothing():
+    """A simulation of a clothing article with the single attribute, color."""
```
Comment (Meltz014) [inline BlueHatGreenHat/clothing.py]: Can you format the attribute here like you did in BodyPart?

File: BlueHatGreenHat/animal.py [hunk 4]
```diff
@@ -51,7 +58,7 @@ def __init__(self):
 
 
 class Animal():
-
+    """A simulation of an animal and its various body parts."""
```
Comment (Meltz014) [inline BlueHatGreenHat/animal.py]: List the attributes

**Review by Meltz014 — CHANGES_REQUESTED** [review]

**Comment by tim-andes** [comment]
Module level docstrings created per Google Style Guide supplemented by PEP257

**Pull request merged**